README.md

```diff
@@ -90,7 +90,11 @@ First install [autopep8](https://pypi.org/project/autopep8/) `$ pip3 install aut
 
 ### Tests
 
-`$ python3 tests/test-integration.py`
+```bash
```

Comment:
I think we'll soon need a script to run all the tests in one go. Maybe `make`?

Reply:
Python has a built-in command to do so:

For us: `python -m unittest discover -s tests/ -p 'test*.py'`